hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/AbstractReplicationTask.java

```diff
@@ -70,6 +70,14 @@ protected AbstractReplicationTask(long containerID,
     this.term = term;
     queued = Instant.now(clock);
   }
+  
+  protected String getMetricName() {
```

Comment:
I would suggest making these abstract methods, which would force any sub-class to implement them or get a compile error. This would reduce the chances of a new type of task of missing these.

Reply:
Thank you @sodonnel .
I've updated it.